userscript-styles/WKED-self-study.css

```diff
@@ -64,6 +64,16 @@
   background-image: none;
 }
 
+#wkof_ds #ss_quiz[data-result="correct"] .answer input {
+  color: var(--ED-text-light, #eeeeee) !important;
+  background-color: var(--ED-correct, #58896f) !important;
+}
+
+#wkof_ds #ss_quiz[data-result="incorrect"] .answer input {
+  color: var(--ED-text-light, #eeeeee) !important;
+  background-color: var(--ED-incorrect, #9c4644;) !important;
```

Comment:
> Is the `!important` necessary? It seems to work just fine without it.

So weird. I need to look at this again. I really hate over-specifying things with `!important` so I'm sure I had a reason (IIRC the original script had a `!important` somewhere that I needed to match).

Reply:
Hmm..

On my system, the `!important` is necessary. Without it, it reverts to the `#8c8` version of green from the self-study quiz styling instead of `--ED-correct`. Same thing for `#f03` and `--ED-incorrect`.

Not sure why it seems to work fine without it for you.

Reply:
alright, I will be keeping the `!important` in.

However I will b eremoving the ; from inside the brackets at the line 74. It looks to be a typo that is breking the style on my side.
```suggestion
  color: var(--ED-text-light, #eeeeee) !important;
  background-color: var(--ED-correct, #58896f) !important;
}

#wkof_ds #ss_quiz[data-result="incorrect"] .answer input {
  color: var(--ED-text-light, #eeeeee) !important;
  background-color: var(--ED-incorrect, #9c4644) !important;
```